meson.build

```diff
@@ -529,7 +535,7 @@ elif get_option('docs') == 'docgen'
     native: true,
     fallback: ['gi-docgen', 'dummy_dep'],
   )
-  gidocgen = find_program('gi-docgen', required: get_option('introspection'))
+  gidocgen = find_program('gi-docgen', required: introspection.allowed())
```

Comment:
`introspection is a feature option, why not use it directly there?`

Reply:
That my inclination but I wasn't sure if the `.disable_auto_if` gets saved into the feature object or it was stateful to the only time I ran it.

Reply:
`get_option('introspection')` will always return the original unmodified feature object. Use of `.disable_auto_if()` creates a cloned copy, and saves that cloned copy for later use any time `introspection` is referenced.

Either way the .allowed() would affect the same thing that find_program(..., required: XXX) would look at.

Reply:
Ah got it, thanks!